README.md

```diff
@@ -72,9 +72,14 @@ And please give some love to our featured sponsors 🤩:
 
 ## Setup
 
-`graphile-migrate` uses two databases in development mode. A development database and 
-a "shadow" database. The "shadow" database is used by `graphile-migrate` to test the 
-consistency of the migrations. Only a single database is used in production environments.
+In development, `graphile-migrate` uses two databases: the main database and a
+"shadow" database. The "shadow" database is used internally by
+`graphile-migrate` to test the consistency of the migrations and perform various
+other tasks.
+
+In production, you'll likely only need to run `graphile-migrate migrate` which
```

Comment:
I'm skeptical to this phrasing `you'll likely`. It creates doubt, especially to those that have no clue what they are supposed to be doing.

Reply:
Yeah; I hear you. That said, it's not guaranteed that you'll only want to run migrate - in rare circumstances or setups you might want to run `status` or other commands. Would saying "most users" instead clarify? It basically means the same, but the semantics may be clearer.

Reply:
That is a great change, semantics matter do matter, but this is heavily debated